www/book.css

```diff
@@ -31,9 +31,15 @@ header, h1, strong {
 }
 }
 h1 { font-size: 105%; margin: 1.5em 0 0; }
+
+.main h1::before { content: none !important; }
```

Comment:
What is this rule for?

Reply:
On the main page (`index.html`), we don't want to auto-number the headings.

Reply:
Please add a comment saying that then.